plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java

```diff
@@ -188,8 +188,7 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 			throw new CoreException(new Status(IStatus.ERROR, CDKCoreActivator.PLUGIN_ID, "Call to minishift up has failed."));
 		}
 
-		IProcess process = addProcessToLaunch(p, launch,s);
-		linkTerminal(p);
```

Comment:
agree. Looks like `linkTerminal` must be called after the [line 85](https://github.com/jbosstools/jbosstools-openshift/pull/1523/files#diff-85015c40a11c890f9f061c7051453feeR85)